strings/names/first.txt

```diff
@@ -687,7 +686,6 @@ Jerrie
 Jerry
 Jesse
 Jessica
-Jesus
```

Comment:
268000 Jesus, It's a REALLY big name in hispanic cultures! Not a Christian reference :)